frontend/src/app/account/account.component.ts

```diff
@@ -1,10 +1,22 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { map } from 'rxjs/operators';
+
+import { AuthService } from '../auth.service';
 
 @Component({
   selector: 'volontulo-account',
   templateUrl: './account.component.html',
 })
 
-export class AccountComponent {
+export class AccountComponent implements OnInit {
```

Comment:
can we add here some kind of formatting? Remember, that We write code to be easly read, not necessarily easly written.

For example it would be a great idea here to separate `constructor` from attribute definition by one line.

BTW don't we have `Organization` type defined? shouldn't it be:
```
organizations$: Observable<Organization[]>;
```
?